tools/configtxgen/encoder_test.go

```diff
@@ -0,0 +1,1674 @@
+/*
```

Comment:
What are the new changes in this file compared to the older encoder_test.go other than the following?

1. removal of dot imports.
2. introduction of a few const variables.
3. changes to the mspDir and cert paths